src/models/layerMetadata/layer3DMetadata.ts

```diff
@@ -35,6 +35,8 @@ export interface ILayer3DMetadata {
   maxFlightAlt: number | undefined;
   geographicArea: string | undefined;
   productBoundingBox: string | undefined;
+  sensorType: SensorType[] | undefined; //sensors
```

Comment:
Shouldn't it be:
sensors: string[] | undefined 
region: string[] | undefined

Reply:
we didn't won't to break other types compatibility - so we changed only our data